ReleaseVersion.props

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Because this file appears to be used only in `SetEnv.targets`, I don't think it passes the [reused abstraction rule](http://codemanship.co.uk/parlezuml/blog/?postid=934) yet. Would it make sense to move this to the existing `Common.targets`?